surfsense_backend/app/connectors/jira_connector.py

```
@@ -0,0 +1,488 @@
+"""
+Jira Connector Module
+
+A module for retrieving data from Jira.
+Allows fetching issue lists and their comments, projects and more.
+"""
+
+import base64
+from datetime import datetime
+import json
```

Ruff (0.12.2)

`json` imported but unused

Remove unused import: `json`

(F401)